core/src/main/java/GameObjects/UrlBuilderWrapper.java

```diff
@@ -0,0 +1,70 @@
+package GameObjects;
+
+/**
+ * Created by Chris on 11/12/2016.
+ */
+
+public interface UrlBuilderWrapper {
+  enum CategoryType {
+    GENERAL(9),
+    FILM(11),
+    MUSIC(12),
+    VIDEOGAMES(15),
+    BOARDGAMES(16),
+    SCIENCENATURE(17),
+    HISTORY(23);
+
+    private int val;
+
+    CategoryType(int val) {
+      this.val = val;
+    }
+
+    public String getVal() {
+      return Integer.toString(val);
+    }
+  }
+
+  enum DifficultyType {
+    EASY(0),
+    MEDIUM(1),
+    HARD(2);
```

Comment:
Leave the underlying value (0,1,2) unspecified since those numbers have no real significance to the type (in the real world or in the url), do away with the 'val' field and constructor, then case 'this' with EASY, MEDIUM, and HARD in getVal().